src/parse/frenchNationalIdFields.js

```diff
@@ -59,7 +59,7 @@ module.exports = [
   },
   Object.assign({}, documentNumberTemplate, {
     line: 1,
-    start: 7,
+    start: 0,
```

Comment:
why this change? The 3 fields are separated in the description of wikipedia. They can always be reconcatenated later

Reply:
https://www.consilium.europa.eu/prado/en/FRA-BO-02002/image-7890.html
First 12 symbols represent complete Document Number. I don't think there is a case to use just part of it somewhere. Without this change all library clients should check if it's France then concatenate 3 fields to get the doc number.

I can make a pull request without this change if you don't want to break the backward compatibility.

Reply:
Thanks for the explanation. This LGTM then. No problem, we'll make a major release with all the changes.